package exceptions;

public class finialBlock {

	public static void main(String[] args) {
		System.out.println("Program started");

		int a = 100;
		try {
			System.out.println(a / 0);

		} catch (Exception e) {
			System.out.println("Enter into catch block");

		} finally {
			System.out.println("Enter into final block");
		}
		System.out.println("Program ended");

	}

}
